### PR TITLE
Collection

### PR DESCRIPTION
**Pull request by Codillac**
The task from module 6.3: Test of the class which has a metod, that picks only the Even numbers from ArrayList<Integer> collection.

### CODE REVIEW

File: kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        numbers = oddNumbersExterminator.exterminate(numbers);
+        //Then
+        boolean areThereAnyOdds = false;
+        for(Integer currentNumber : numbers){
+            if(currentNumber % 2 != 0){
+                areThereAnyOdds = true;
+            }
+        }
+        Assert.assertFalse(areThereAnyOdds);
+    }
```
Comment (MARudnicki) [inline kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java]: j.w.

File: kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
```
Comment (MARudnicki) [inline kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java]: Zbedne komplikownie kodu. Znacznie latwiej to zapisac:

for(Integer currentNumber : numbers){
Assert.assertFalse(currentNumber % 2 != 0);
}

**Review by MARudnicki — CHANGES_REQUESTED** [review]
Testy do rector'u 

Uzywaj na przyszlosc prosze nazw dla branchy zawierajacych numery z kodilli. W stylu 6.3 albo task6.3

File: kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java (hunk 2 above)
Comment (Codillac) [reply kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java]: Thank you for your valuable remarks. I have done the changes and commited them already.

**Pull request merged**